server/src/index.ts

```diff
@@ -79,7 +78,7 @@ class HttpException extends Error {
     this.message = message;
   }
 }
-
+/* tslint:disable:no-unused-variable */
```

Comment:
Please remove the unused variable or replace with `_` instead of disabling the linter.